frontend/public/components/markdown-view.tsx

```diff
@@ -17,14 +17,18 @@ type ShowdownExtension = {
   replace?: (...args: any[]) => string;
 };
 
-const markdownConvert = (markdown, extensions: ShowdownExtension[]) => {
+const markdownConvert = (markdown, extensions: ShowdownExtension[], options: ShowdownOptions) => {
```

Comment:
We need to set a default value for the  `options`, otherwise the `Object.entries` below will error out

Reply:
Good catch, @jhadvig. Fixed.